imports/client/components/SpinnerTimer.tsx

```diff
@@ -87,6 +87,7 @@ const SpinnerTimer = ({
       ref={canvasRef}
       width={width}
       height={height}
+      style={{ alignSelf: "center"}}
```

Comment:
Hmmm, I think this would make more sense in `NotificationCenter.tsx` as a 

```typescript
const CenteredSpinnerTimer = styled(SpinnerTimer)`
  align-self: center;
`;
```

And then swapping out the callsite.  This component itself is trying to implement the spinner generically; positioning/aligment is really more up to the caller.